server/src/streaming/systems/stats.rs

```diff
@@ -45,7 +49,9 @@ impl System {
                 .map(|s| {
                     s.topics
                         .values()
-                        .map(|t| t.consumer_groups.len() as u32)
+                        .map(|t: &crate::streaming::topics::topic::Topic| {
```

Comment:
Make it shorter and move `crate::streaming::topics::topic::Topic` to `use` block